src/Umbraco.Core/HealthChecks/Checks/Security/HstsCheck.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Umbraco.
+// See LICENSE for more details.
+
+using Umbraco.Core.Services;
+using Umbraco.Web;
+
+namespace Umbraco.Core.HealthChecks.Checks.Security
+{
+    /// <summary>
+    /// Health check for the recommended production setup regarding the Strict-Transport-Security header.
+    /// </summary>
+    [HealthCheck(
+        "E2048C48-21C5-4BE1-A80B-8062162DF124",
+        "Cookie hijacking and protocol downgrade attacks Protection (Strict-Transport-Security Header (HSTS))",
+        Description = "Checks if your site, when running with HTTPS, contains the Strict-Transport-Security Header (HSTS). If not, it adds with a default of 18 weeks.",
```

Comment:
We say that we add the HSTS header if it's not there, but as I understand, we don't do that anymore, but just link to documentation.